scripts/gitlab/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM nvidia/cuda:10.0-devel-ubuntu18.04
+RUN apt-get update && apt-get install -y \
+        bc \
+        wget \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install CMake
+RUN export CMAKE_VERSION=3.13.4 && \
```

Comment:
Could be just use Ubuntu's cmake?

Reply:
Yes if we stick to CUDA 10.0
But say you only have CUDA 9.2 on your machine, then you'd have to change the base image to `nvidia/cuda:9.2-devel-ubuntu16.04` (18.04 is not available) which comes with cmake-3.5.1 which is too old for Cabana.  I did not bother making the base image a build argument but it is intended as a customization point.